package.json.in

```diff
@@ -54,6 +54,7 @@
         "@opam-alpha/ocaml-migrate-parsetree": "*",
         "@opam-alpha/result": "*",
         "@opam-alpha/topkg": "0.8.1",
+        "@opam-alpha/ppx_tools_versioned": "5.0alpha",
```

Comment:
Please use 5.0.0 here. 5.0 is not a valid semantic version and we converted the package as version "5.0.0": https://github.com/npm-opam/ppx_tools_versioned/blob/latest/package.json